progress/angular/projects/ng-progress-icons/README.md

```diff
@@ -1,41 +0,0 @@
-# PX Blue Progress Icons
```

Comment:
This is the readme that actually gets published with the NPM package. We need to either keep it, or add a post-build step to copy the other readme into the /dist/ng-progress-icons folder

Reply:
Copied the /progress/angular/README.md to the dist folder